portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final ArrayList<String> COMMENTS = new ArrayList<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Kevin!</h1>");
+    initializeComments();
+    response.setContentType("application/json");
+    response.getWriter().println(convertCommentsToJson());
+  }
+
+  private void initializeComments() {
+    COMMENTS.add("Hello!");
+    COMMENTS.add("Amazing website!");
+    COMMENTS.add("Google SPS!");
   }
+
+  private String convertCommentsToJson() {
+    Gson gson = new Gson();
```

Comment:
the Gson object can be put as class variable

Reply:
Done